automation/taskcluster/decision_task.py

```diff
@@ -194,6 +194,28 @@ def pr_or_push(artifacts_info):
     return (build_tasks, other_tasks)
 
 
+def push(artifacts_info):
+    if SKIP_TASKS_TRIGGER in PR_TITLE:
+        print("Push title contains", SKIP_TASKS_TRIGGER)
+        print("Exit")
+        exit(0)
+
+    modules = [_get_gradle_module_name(artifact_info) for artifact_info in artifacts_info]
+
+    build_tasks = {}
+    other_tasks = {}
+
+    for module in modules:
+        tasks = create_module_tasks(module)
+        for task in tasks:
+            build_tasks[taskcluster.slugId()] = task
+
+    for craft_function in (BUILDER.craft_detekt_task, BUILDER.craft_ktlint_task, BUILDER.craft_compare_locales_task, BUILDER.craft_ui_tests_task):
```

Comment:
I see `push()` generates the same tasks than `pr()`, with the only exception of `BUILDER.craft_ui_tests_task` being added.

I would suggest to modify `push` to something like this:

```py
def push(artifacts_info):
    all_tasks = pr(artifacts_info)
    other_tasks = all_tasks[1]
    other_tasks[taskcluster.slugId()] = BUILDER.craft_ui_tests_task()
    return all_tasks
```

What do you think?

Reply:
@JohanLorenzo i think this is really cool!  didn't know I could do this before, updating now....